terraform/hydra-projects.tf

```diff
@@ -69,3 +69,62 @@ resource "hydra_project" "simple_nixos_mailserver" {
     value = "https://gitlab.com/simple-nixos-mailserver/nixos-mailserver"
   }
 }
+
+resource "hydra_project" "nixpkgs" {
+  name         = "nixpkgs"
+  display_name = "nixpkgs"
+  description  = "you know what this is"
+  homepage     = "https://github.com/NixOS/nixpkgs"
+  owner        = "admin"
+  enabled      = true
+  visible      = true
+}
+
+resource "hydra_jobset" "nixpkgs_cuda" {
+  project     = hydra_project.nixpkgs.name
+  state       = "enabled"
+  visible     = true
+  name        = "cuda"
+  type        = "legacy"
+  description = "Testing CUDA support. Come help the CUDA team! https://nixos.org/community/teams/cuda/"
+
+  nix_expression {
+    file  = "pkgs/top-level/release-cuda.nix"
```

Comment:
Uh-huh, we'd have to update that in Nixpkgs, this currently just lists a small and arbitrary selection of packages

Reply:
Yeah, that will be up to you. I was wondering the same about the supportedPlatforms. Maybe that's something you want to control on the other end as well?

Reply:
Well supported platforms are just `*-linux` at this point, I think it's ok to keep that constant